app/views/challenge_claims/_unposted_claim_blurb.html.erb

```diff
@@ -35,7 +35,7 @@
     </p>
 
     <div class="icon">
-      <%= image_tag(claim.collection.icon.url(:standard), :size => "100x100", :alt => claim.collection.icon_alt_text, :class => "icon", skip_pipeline: true) %>
+      <%= image_tag(standard_icon_url(claim.collection), size: "100x100", alt: claim.collection.icon_alt_text, class: "icon", skip_pipeline: true) %>
```

Comment:
Because this is always the same image tag, would it make sense pull that into the method? Similar to the other image_tag helper methods that this PR modifies.

Okay well, the two in the form and header don't have the icon class. But that looks more like a bug.